app/screens/Room/index.js

```diff
@@ -744,7 +744,7 @@ class Room extends Component {
           renderNavigationView={this.renderRoomInfo}
           keyboardDismissMode="on-drag">
               {isLoadingMore ? this.renderLoading(40) : null}
-              {isLoadingMessages ? this.renderLoading() : this.renderListView()}
+              {this.renderListView()}
```

Comment:
Why? Instead of rendering spinner while no data to render list view, you render "Failed to load" message.